docs/pages/material-ui/getting-started/templates/blog.js

```diff
@@ -1,11 +1,14 @@
 import * as React from 'react';
 import AppTheme from 'docs/src/modules/components/AppTheme';
```

Comment:
Could we rename this? We have two `AppTheme` in the React tree, and this component has nothing to do with theming.

Reply:
Actually, we could add a title to those pages, this could be a better DX. I imagine we should also remove things like `description="A onepirate theme"` what's the point, the page is `<meta name="robots" content="noindex,nofollow" />` it's absurd.

Reply:
> Could we rename this? We have two `AppTheme` in the React tree, and this component has nothing to do with theming.

Thanks for the feedback but I would do it in a separate PR.